TorchSharp.PyBridge/TorchSharp.PyBridge.csproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <PackageReference Include="Razorvine.Pickle" Version="1.5.0" />
     <PackageReference Include="TorchSharp" Version="[0.101.3,)" PrivateAssets="All" />
```

Comment:
Maybe consider removing `PrivateAssets` so it will flow into dependency graph and user can get warning message during compiling time about package version mismatch if they already target on a different version torchsharp?

Reply:
I considered that, my reason against was because this is just an extension methods package, I didn't want someone who installs it to also get any specific version of TorchSharp, but rather it should just work with whatever he was using anyway. 
I'm open to change it though, let's discuss for the next version.

Reply:
Yeah I understand, the reason why I suggest removing `PrivateAssets` is because it's not quite straightforward to figure out the right latest `torchsharp` version that is supported by `Torchsharp.Bridge` when user refers to a package before `0.101.3`. They might need to visit this `.csproj` to find the right version

Reply:
I get a compilation error which there is a target mismatch: 
![image](https://github.com/shaltielshmid/TorchSharp.PyBridge/assets/20192446/91bfe2f6-af42-41d8-9750-c159a0d00d83)

Reply:
Ha, OK. I might not look closely enough. I tried another time and it seems that the version information is indeed contains in the error message.